providers/asrockrack/firmware.go

```diff
@@ -74,7 +74,16 @@ func (a *ASRockRack) firmwareInstallBMC(ctx context.Context, reader io.Reader, f
 	a.log.V(2).WithValues("step", "2/4").Info("upload BMC firmware image")
 	err = a.uploadFirmware(ctx, "api/maintenance/firmware", reader, fileSize)
 	if err != nil {
-		return errors.Wrap(err, "failed in step 2/4 - upload BMC firmware image")
+		// Try the special URL for e3c256d4i
```

Comment:
Can we check what the error is - if it contains 404, then we attempt the other endpoint.

Reply:
We use the model now